core/dbt/events/functions.py

```diff
@@ -144,16 +144,20 @@ def scrub_dict_secrets(values: Dict) -> Dict:
 # the usage site.
 def event_to_dict(e: T_Event, msg_fn: Callable[[T_Event], str]) -> dict:
     level = e.level_tag()
-    return {
+    event_dict = {
+        'type': 'log_line',
         'log_version': e.log_version,
         'ts': e.get_ts(),
         'pid': e.get_pid(),
         'msg': msg_fn(e),
         'level': level,
         'data': Optional[Dict[str, Any]],
-        'event_data_serialized': True
+        'event_data_serialized': True,
```

Comment:
what is `event_data_serialized`? I saw it in a different PR and removed it. Maybe I shouldn't have.

Reply:
#4326 removes the need for it.

Reply:
that's the PR I removed it in so that's good lol.